src/providers/export/azureCustomVision.ts

```diff
@@ -38,6 +39,24 @@ export enum NewOrExisting {
     Existing = "existing",
 }
 
+/**
+ * Azure regions
+ */
+export enum AzureRegion {
```

Comment:
we're not using elsewhere, but I can see this bubbling up at some point, outside of CV.

Reply:
Agreed - will move it up at that time.